msteams-platform/task-modules-and-cards/cards/cards-reference.md

```diff
@@ -587,7 +587,7 @@ Bot Framework reference:
 
 Card collections are supported in Teams.
 
-Card collections are provided by the Bot Framework: `builder.AttachmentLayout.carousel` and `builder.AttachmentLayout.list`. These collections can contain Adaptive, Hero, or Thumbnail cards.
+Card collections: `builder.AttachmentLayout.carousel` and `builder.AttachmentLayout.list`. These collections can contain Adaptive, Hero, or Thumbnail cards.
```

Comment:
Confirm if the statement you have deleted is not required. If it is required to be mentioned then add it as a sentence to the text.
```Suggestion
Card collections: `builder.AttachmentLayout.carousel` and `builder.AttachmentLayout.list`. The Bot Framework provides the Card collections. These collections contain Adaptive, Hero, or Thumbnail cards.
```

Reply:
Confirmed with SME that  " The Bot Framework provides the Card collections." is not required. So, removing this line. Accepted rest of the suggestion.